main.go

```diff
@@ -20,6 +20,7 @@ var (
 	upstreamServers = flag.String("s", "one.one.one.one:853@1.1.1.1,dns.google:853@8.8.8.8", "comma-separated list of upstream servers")
 	logPath         = flag.String("l", "", "log file path")
 	isLogVerbose    = flag.Bool("v", false, "verbose mode")
+	evictMetrics    = flag.Bool("em", false, "collect metrics on evictions")
```

Comment:
Please update README, section Usage.

Reply:
Done, even if copy-pasting the output of the "-help" flag in the readme seems quite redundant.